httpbp/middlewares.go

```diff
@@ -158,3 +160,31 @@ func InjectEdgeRequestContext(truster HeaderTrustHandler, impl *edgecontext.Impl
 		}
 	}
 }
+
+// SupportedMethods returns a middleware that checks if the request is made
+// using one of the given HTTP methods.
+//
+// Returns a raw, plain text 405 error response if the method is not supported.
+func SupportedMethods(method string, additional ...string) Middleware {
```

Comment:
🔕 Is the separation of `method` and `additional` for forcing at least one arg passed in?

Reply:
Yes